v2.6/usage/decommissioning-a-node.md

```diff
@@ -11,9 +11,15 @@ redirect_from: latest/usage/decommissioning-a-node
 - You are receiving an error about an IP address already in use.
 - Hosts are regularly added and removed from your cluster.
 
+### Why you might not be interested in this guide
+
+- You are running Calico in Kubernetes-Datastore Mode, which automatically
+handles node deletion.
+- You are running the Calico Policy Controller v1.0.1+, which automatically handles node deletion.
```

Comment:
I think we should re-phrase this in a positive way. Rather than "Why you're not interested in this" it should be "When should you read this document"

The reasons shouldn't talk about the policy controller (it's the node controller that does this) and it really should talk more in terms of user-facing components, so "kube-controllers" is really appropriate.

Users will also need to follow this if not using self-hosted or another approach which populates the k8s node references.